.editorconfig

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2011-2018 Genestack Limited
+# All Rights Reserved
+# THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF GENESTACK LIMITED
+# The copyright notice above does not evidence any
+# actual or intended publication of such source code.
+#
```

Comment:
This repo is MIT licensed, [see this notice, for example](https://github.com/genestack/JS-SDK/blob/master/packages/interfaces/src/entry-point.ts#L1-L6)

Reply:
Wow, I'll fix It soon.

Reply:
@RB-Lab, done. Have I fixed It correct?